cadc-vos-server-db/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id 'java'
+    id 'maven'
+    id 'maven-publish'
+    id 'checkstyle'
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+apply from: '../opencadc.gradle'
+
+sourceCompatibility = 1.8
+
+group = 'org.opencadc'
+
+version = '1.3.4'
+
+description = 'OpenCADC VOSpace server library DB'
+def git_url = 'https://github.com/opencadc/vos'
+
+dependencies {
+    compile 'org.opencadc:cadc-util:[1.6,2.0)'
+    compile 'org.opencadc:cadc-vos:[1.2.1,)'
```

Comment:
this should have to depend on cadc-vos-server explicitly... curious why it builds without that

Reply:
so after above fix, cadc-vos-server-db depends on cadc-vos-server and not the other way around;

then cavern will not include the -db code at all